pkgs/by-name/co/codex/package.nix

```
@@ -27,6 +27,9 @@ rustPlatform.buildRustPackage (finalAttrs: {
     pkg-config
     openssl
   ];
+  buildInputs = [
+    openssl.dev
+  ];
```

1. `dev` outputs are already added in mkDerivation, no need to specify them.
2. The fix is actually more prosaic than it seems: the previous author incorrectly put `openssl` to `nativeBuildInputs`, when it's a library
```suggestion
  ];
  buildInputs = [
    openssl
  ];
```

---

No, it didn't pass. Just they overlooked it.